### DISCUSSION TITLE
Character size is larger than input text

### DISCUSSION THREAD

**Discussion by cscomfort** [starship/starship#5180] [Q&A]
Just getting started with Starship and trying to sort out why, in my environment, the [character](https://starship.rs/config/#character) symbol `❯` appears disproportionately larger than the adjacent input text. I'm using the FiraCode Nerd Font and the default, out-of-the box configuration (i.e., no custom `.toml`). See the screenshot below. Based on the example and what was provided in the [What is the configuration used in the demo GIF?](https://starship.rs/faq/#what-is-the-configuration-used-in-the-demo-gif) FAQ, it looks like the character is smaller than what presents in my environment.

Any thoughts will be appreciated.

Additional info:
* Latest version of Starship installed with Homebrew
* Apple M2 / OS X 13.3
* iTerm2 (Build 3.4.19)

<img width="974" alt="Screenshot 2023-05-11 at 10 43 58 AM" src="https://github.com/starship/starship/assets/18727632/c16d8d95-dab5-4e57-a5b4-5afc2d07f40f">

**Comment by mickimnet** [comment]
I just tried to replicate the behavior (on _macOS_ 13.3.1 with _iTerm2_ 3.5.0beta10) and I have no clue. See for yourself:
<img width="652" alt="image" src="https://github.com/starship/starship/assets/2911922/02b2253f-1eed-4981-8a61-543bcf85c374">
<img width="586" alt="image" src="https://github.com/starship/starship/assets/2911922/c892f28b-623e-4944-b96b-4a543ab2f9f7">
What _NerdFont_ version of _FiraCode_ are you using?

**Comment by cscomfort** [comment]
I got the font files directly from the [Nerd Fonts download page](https://www.nerdfonts.com/font-downloads), so not sure what the issue may have been. However, after removing all of the fonts and then re-installing with Homebrew all is looking closer to what I would expect.

Thanks for taking the time to respond to the thread -- cheers!
